cli/flox-rust-sdk/src/models/environment/generations.rs

```diff
@@ -1746,6 +1748,36 @@ mod tests {
             assert_eq!(*actual, expected)
         }
 
+        #[test]
+        fn parse_v2_invalid() {
+            let metadata = json!({
+                "version": 2,
+                "unknown_field": true,
```

Comment:
Suggestion: Unknown fields should be parsed correctly (as other fields), I think we should rather make clear that _required fields_ are missing and causing the parsing issue.